Apps/Common/src/Api/IKeycloakAdminApi.cs

```diff
@@ -33,7 +33,7 @@ public interface IKeycloakAdminApi
         /// <param name="token">The bearer token to authorize the call.</param>
         /// <returns>A resulting UserRepresentation object.</returns>
         [Get("/users?briefRepresentation=true&username={username}&exact=true")]
-        Task<UserRepresentation> GetUserAsync(string username, [Authorize] string token);
+        Task<List<UserRepresentation>> GetUserAsync(string username, [Authorize] string token);
```

Comment:
Would recommend renaming to `GetUsersByUsernameAsync` since it's returning a collection.